doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='asynchronous-signal-events-con']
+
+= Asynchronous signal events
+
+In some cases where several process instances from different process definitions are waiting for the same signal. They are generally executed sequentially in the same single thread. But, if one of those process instances throws a runtime exception, all the other process instances are affected and usually result in a rolled back transaction. To avoid this situation, {PRODUCT} supports using asynchronous signals events for:
+
+* Throwing intermediate signal events
+* End events
+
+In the process designer of {CENTRAL}, set the data input value of the throw event to *async* in the *Data assignments* section of *Properties* panel leaving other values blank. It will automatically set the executor service on each session. This ensures that each process instance is signaled in a different transaction.
```

Comment:
Suggested rewrite:
To ensure that each process instance is signaled to a different transaction, set the data input value of the throw event as async in the Data assignments of the Properties panel. This sets the executor service in each session.

Reply:
Done